sdk/storage/storage-queue/src/MessageIdClient.ts

```diff
@@ -3,7 +3,7 @@
 
 import { TokenCredential, isTokenCredential, isNode } from "@azure/core-http";
 import * as Models from "./generated/lib/models";
-import { Aborter } from "./Aborter";
+import { AbortSignal } from "@azure/abort-controller";
```

Comment:
In our other SDKs we're using the `AbortSignalLike` interface from `@azure/abort-controller` as the type for `abortSignal`, instead of the `AbortSignal` class. This is so customers can also use the native AbortController/AbortSignal built into modern web-browsers as well.

Reply:
Here's an example in `event-hubs`: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/src/receiver.ts#L272